README.md

```diff
@@ -6,40 +6,85 @@ A Docker LogSpout image to stream logs from your containers to LogDNA.
 
 * Initial Release
 
-
 ## How to Use
 
 ### Environment Variables
-* __LOGDNA_KEY__: LogDNA Ingestion Key, *Required*
+
+The following variables can be used to tune `LogSpout` for specific use cases.
+
+#### Log Router Specific
+
 * __FILTER_NAME__: Filter by Container Name with Wildcards, *Optional*
 * __FILTER_ID__: Filter by Container ID with Wildcards, *Optional*
-* __FILTER_SOURCES__: Filter by Comma-separated List of Sources, *Optional*
-* __FILTER_LABELS__: Filter by Comma-separated List of Labels, *Optional*
+* __FILTER_SOURCES__: Filter by Comma-Separated List of Sources, *Optional*
+* __FILTER_LABELS__: Filter by Comma-Separated List of Labels, *Optional*
+
+__Note__: More information can be found [here](https://github.com/gliderlabs/logspout/tree/0da75a223db992cd5abc836796174588ddfc62b4/routesapi#routes-resource).
+
+#### Ingestion Specific
+
+* __LOGDNA_KEY__: LogDNA Ingestion Key, *Required*
 * __HOSTNAME__: Alternative Hostname, *Optional*
-* __TAGS__: Comma-separated List of Tags, *Optional*
+* __TAGS__: Comma-Separated List of Tags, *Optional*
 * __LOGDNA_URL__: Specific Endpoint to Stream Log into, *Optional*
+  * __Default__: `logs.logdna.com/logs/ingest`
+* __VERBOSE__: Enabling or Disabling to Log `LogSpout` Container, *Optional*
+  * __Default__: Enabled
+  * __Note__: `0` to Disable
+
+__Note__: Logging the `LogSpout` Container is recommended to keep track of HTTP Request Error or Exceptions.
+
+#### Related to HTTP Client
+* __DIAL_KEEP_ALIVE__: The interval (in `seconds`) between keep-alive probes for an active network connection.
+  * __Default__: 60
+  * __Source__: [net/dial.go#Timeout](https://github.com/golang/go/blob/master/src/net/dial.go#L72-L79)
+* __DIAL_TIMEOUT__: The maximum amount of time (in `seconds`) a dial will wait for a connect to complete.
+  * __Default__: 60
+  * __Source__: [net/dial.go#Timeout](https://github.com/golang/go/blob/master/src/net/dial.go#L27-L39)
+* __IDLE_CONN_TIMEOUT__: The maximum amount of time (in `seconds`) an idle (keep-alive) connection will remain idle before closing itself.
+  * __Default__: 60
+  * __Source__: [net/http/transport.go#IdleConnTimeout](https://github.com/golang/go/blob/master/src/net/http/transport.go#L213-L217)
+* __HTTP_CLIENT_TIMEOUT__: Time limit (in `seconds`) for requests made by this HTTP Client, *Optional*
+  * __Default__: 60
+  * __Source__: [net/http/client.go#Timeout](https://github.com/golang/go/blob/master/src/net/http/client.go#L89-L104)
+* __TLS_HANDSHAKE_TIMEOUT__: The maximum amount of time (in `seconds`) allowed to wait for a TLS handshake, *Optional*
+  * __Default__: 30
+  * __Source__: [net/http/transport.go#TLSHandshakeTimeout](https://github.com/golang/go/blob/master/src/net/http/transport.go#L171-L173)
+
+#### Limits
+* __FLUSH_INTERVAL__: How frequently batches of logs are sent (in `milliseconds`), *Optional*
+  * __Default__: 250
+* __INACTIVITY_TIMEOUT__: The timeout to detect failure in `Docker API` and restart, *Optional*
```

Comment:
How long to wait for inactivity before declaring failure in the `Docker API` and restarting,

Reply:
I liked this!